### PR TITLE
Update the-unarchiver to 3.11.4

### PR DESCRIPTION
**Pull request by tiiiecherle**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/the-unarchiver.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 cask 'the-unarchiver' do
-  version '3.11.3,115:1507740941'
-  sha256 '04dc20b845171e25c7137e504cd5a5387a81bdfdc51d2ad810d5fdcf84cff8c6'
+  version '3.11.4'
+  sha256 '0b65ed1af57766c9ce4db9207390f31ffc3a2c1c274616ac20749739661e4929'
 
   # devmate.com/cx.c3.theunarchiver was verified as official when first introduced to the cask
-  url "https://dl.devmate.com/cx.c3.theunarchiver/#{version.after_comma.before_colon}/#{version.after_colon}/TheUnarchiver-#{version.after_comma.before_colon}.zip"
```
Comment (commitay) [inline Casks/the-unarchiver.rb]: Please use the full URL.

**Review by commitay — DISMISSED** [review]

**Comment by tiiiecherle** [comment]
I´m sorry. I don`t know where to find the last number in the version.
old      version '3.11.3,115:1507740941'
new	   version '3.11.4,116:???'

**Comment by commitay** [comment]
`3.11.4,116:1520347991`

`https://dl.devmate.com/cx.c3.theunarchiver/116/1520347991/TheUnarchiver-116.zip`

**Comment by tiiiecherle** [comment]
thanks

**Pull request merged**